tests/cypress/integration/devportal/000-general/03-self-signup.spec.js

```diff
@@ -215,5 +216,4 @@ describe("Self SignUp", () => {
         cy.updateTenantConfig(tenantAdminUsername, tenantAdminPassword, testTenant, tenantConfigJson);
         cy.enableSelfSignUpInCarbonPortal(tenantAdminUsername, tenantAdminPassword, testTenant);
     })
-
 });
```

Comment:
Please add the missing new line

Reply:
Fixed in https://github.com/wso2/apim-apps/pull/454